cluster/juju/layers/kubernetes-master/templates/addons/dashboard-controller.yaml

```diff
@@ -0,0 +1 @@
+# Empty placeholder template, to be replaced at build time
```

Comment:
You can have the tactic add additional files, you just have to make sure it also signs them in the manifest.  It's very straightforward; see [DynamicHookBind](https://github.com/juju/charm-tools/blob/master/charmtools/build/tactics.py#L291) as an example.